manager/ui/war/plugins/api-manager/ts/api/api-contracts.ts

```diff
@@ -30,16 +30,66 @@ _module.controller("Apiman.ApiContractsController",
             })
         });
 
-        $scope.approveContract = (contract: ContractSummaryBean) => {
+        let handleApprovalError = function (err) {
+            if (err !== 'cancel') {
+                console.error(err)

```

Comment:
Please consider using `Logger` for this. Looks like it's already injected, so should be usable directly.

Logger.debug or something